inc/html.class.php

```diff
@@ -1860,7 +1860,7 @@ private static function getPageHeaderTplVars(): array {
          'is_debug_active'       => $_SESSION['glpi_use_mode'] == Session::DEBUG_MODE,
          'is_impersonate_active' => Session::isImpersonateActive(),
          'founded_new_version'   => $founded_new_version,
-         'language_name'         => Dropdown::getLanguageName($_SESSION['glpilanguage']),
+         'language_name'         => User::showSwitchLangForm(),
```

Comment:
Maybe it should be replaced by `{{ call('User::showSwitchLangForm') }}` in template. Passing the whole dropdown code in a variable seems weird to me.